astropy/io/ascii/tests/test_rst.py

```diff
@@ -132,12 +132,12 @@ def test_trailing_spaces_in_row_definition():
     """ Trailing spaces in the row definition column shouldn't matter"""
     table = """
 # comment (with blank line above)
-   ==== ==== ====    
+   ==== ==== ====
```

Comment:
Given the comment above, I think the trailing spaces were there on purpose, no? I.e., really, the `nopep8` below should be changed.